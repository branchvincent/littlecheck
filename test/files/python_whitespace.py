#!/usr/bin/python3
import sys

sys.stdout.write("    one\n")
# CHECK: {{^}}    one

sys.stdout.write("    two\n")
# CHECK: two

sys.stdout.write("three\n")
# CHECK:     three

sys.stdout.write("   four\n")
# CHECK: {{^}} four
