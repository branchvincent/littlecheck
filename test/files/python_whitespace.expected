Failure in python_whitespace.py:

  The CHECK on line 14 wants:
    {{^}} four

  which failed to match line stdout:4:
       four

  Context:
        one
        two
    three
       four <= does not match CHECK on line 14: {{^}} four

  when running command:
    /usr/bin/python3 python_whitespace.py
